### PR TITLE
WR-364: updates fixes tweaks

### PR DESCRIPTION
**Pull request by zetagraph**
Various Tickets and Bug Fixes

### CODE REVIEW

File: src/web/modules/custom/workbc_custom/src/Plugin/Block/ExploreCareersBlock.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ public function build() {
 
 
     $options = [];
-    $link = Link::fromTextAndUrl(t($config['label_2'] ?? ""), Url::fromUri('internal:/career-profiles', $options))->toString();
+    $link = Link::fromTextAndUrl(t($config['label_2'] ?? ""), Url::fromUri('internal:/plan-career/explore-careers/career-search', $options))->toString();
```
Comment (infojunkie) [inline src/web/modules/custom/workbc_custom/src/Plugin/Block/ExploreCareersBlock.php:114]: This should not be hard-coded. I understand it's not your code, but I am flagging this as something to fix.

Comment (zetagraph) [reply src/web/modules/custom/workbc_custom/src/Plugin/Block/ExploreCareersBlock.php:114]: Ok, I can create a ticket for it. Who should I assign it to?

**Review by infojunkie — COMMENTED** [review]
Lgtm, except for the hard-coded paths. Let's agree on a fix for those before merging, and this fix can be done is a subsequent ticket.

**Comment by zetagraph** [comment]
ok, merging as there are a lot of "hot fixes" and moving pieces in here unrelated to the paths. Will create a ticket to update the path separately.

**Pull request merged**